packages/terra-doc-template/src/terra-dev-site/doc/doc-template/Placeholder.2.doc.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+import ReadMe from '../../../../docs/README.md';
+import { name } from '../../../../package.json';
+
+/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
+// Component Source
+import PlaceholderSrc from '!raw-loader!../../../../src/Placeholder.jsx';
+
+// Example Files
+import PlaceholderDefault from '../example/PlaceholderDefault';
+import PlaceholderDefaultSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/PlaceholderDefault';
+import PlaceholderLight from '../example/PlaceholderLight';
+import PlaceholderLightSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/PlaceholderLight';
+/* eslint-enable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
+
+const DocPage = () => (
+  <DocTemplate
+    packageName={name}
+    readme={ReadMe}
+    srcPath={`https://github.com/cerner/terra-core/tree/master/packages/${name}`}
+    examples={[
+      {
+        title: 'Default Placeholder',
```

Comment:
Nit: specify dark, although the default is dark
```suggestion
        title: 'Dark Placeholder',
```